internal/locate/region_cache.go

```diff
@@ -483,6 +484,15 @@ func (c *RegionCache) SetPDClient(client pd.Client) {
 	c.pdClient = client
 }
 
+func (c *RegionCache) GetApiVersion() kvrpcpb.APIVersion {
```

Comment:
why not store api version when creating RegionCache?

Reply:
`ApiVersion` should be removed from  `RegionCache`, and moved to `RegionRequestSender`. The API version could be derived while constructing `RegionRequestSender`.